zospy/functions/nce.py

```diff
@@ -33,3 +32,73 @@ def object_change_type(obj: _ZOSAPI.Editors.NCE.INCERow, new_type: constants.Edi
     # Apply
     new_surface_type_settings = obj.GetObjectTypeSettings(new_type)
     obj.ChangeType(new_surface_type_settings)
+
+def find_object_comment(nce: _ZOSAPI.Editors.NCE, comment: str, case_sensitive: bool=False) -> list[int] | int:
+    """Function equivalent to ZPL numeric function OBJC($A).
+
+    Parameters
+    ----------
+    nce: ZOSAPI.Editors.NCE
+        The Non-sequential Component Editor (NCE).
+    comment: str
+        String that is searched for in the Comment column of the NCE.
+    case_sensitive: bool=False
+        Flag that specifies whether the search is case-sensitive or not.
+
+
+    Returns
+    ----------
+    A list of integer indices corresponding to the rows of the NCE
```

Comment:
Specify the type of the returned value here, in the same format as used elsewhere (i.e. [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#returns))

Reply:
Done